src/libraries/System.Formats.Cbor/src/System/Formats/Cbor/HalfHelpers.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.using System;
+
+using System.Buffers.Binary;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace System.Formats.Cbor
+{
+    // Temporarily implements missing APIs for System.Half
+    // Remove class once https://github.com/dotnet/runtime/issues/38288 has been addressed
+    internal static class HalfHelpers
+    {
+        public const int SizeOfHalf = sizeof(short);
```

Comment:
Why not just use `sizeof(Half)` inline where needed?

Reply:
Compiler complains that it doesn't have a predefined size, so would have to mark as unsafe. Presumably because it's a new type?